src/watts/plugin_moose.py

```diff
@@ -162,6 +161,20 @@ def options(self, moose_exec):
         """
         self.moose_exec = moose_exec
 
+    def convert_unit(self, params: Parameters, unit_system, unit_temperature):
```

Comment:
Added this method explicitly to allow the unit test to check that the unit conversion is correct.

Reply:
The method as defined in `Plugin` will be available by inheritance, so it's not necessary to explicitly define it in PluginMOOSE and PluginOpenMC